website/docusaurus.config.js

```diff
@@ -6,17 +6,30 @@ module.exports = {
   baseUrl: "/graphql-link/",
   onBrokenLinks: "ignore",
   onBrokenMarkdownLinks: "warn",
-  favicon: "img/logo.ico",
+  favicon: "img/logo_without_text-removebg-preview.png",
```

Comment:
There is a lot of images added in PR and not every image is used. can we remove those?

Reply:
done